modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -0,0 +1,59 @@
+= OpenID Connect (OIDC)
+:toc: right
+:toclevel: 2
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+:openid-connect-url: https://openid.net/connect/
+:openid-config-url: http://localhost:3000/.well-known/openid-configuration
```

Comment:
nobody will understand localhost:3000 here. We might want to use a placeholder for the IdP.
In any case we should always use https

Reply:
Removed that attribute as it's unused.